packages/dumb/src/components/stats/models.tsx

```diff
@@ -46,3 +46,10 @@ export enum StatLevel {
   Low = 'low',
   VeryLow = 'veryLow'
 }
+
+export interface BgClasses {
+  high: number // High threshold represents all the values between target and high. The veyHigh threshold are all the values greater than high.
```

Comment:
Typo in `veryHigh` (but should it be mentioned, given that it is not defined in the `BgClasses`?)

Reply:
I still prefer to mention it as it has been proven confusing (the fact of veryHigh not being defined here seems confusing, but I won't add it here just for that, so a comment is an ok solution I think)